guess_number/guess.py

```diff
@@ -0,0 +1,71 @@
+import random
+
+random_number = random.randint(0, 100)
+
+
+def guess_number():
+
+    while True:
+        user_guess = int(input("Guess a number: "))
+        if user_guess == random_number:
+            print("Just right!")
+            return random_number
+            break
+
+        elif user_guess < random_number:
+            print("Too low")
+
+        elif user_guess > random_number:
+            print("Too high")
+
+
+print(guess_number())
+
+
+# An alternative to the guessing game: have the computer guess the number
+secret_number = 6
+high = 10
+low = 0
+guess = (high + low) // 2
+
+
+while True:
+    print(f"Is your guess {str(guess)}?")
```

Comment:
I believe an f-string will automatically coerce an `int` to a string, so there's probably no need for the `str` call.